### PR TITLE
fix(Edit Image Node): Fix Text operation by setting Arial as default font

### PR DESCRIPTION
**Pull request by ayatnkw**
## Summary

The Edit Image node's text operation doesn't work because the default font is not set. This PR fixes this by setting the default font to Arial if it exists in the system fonts. 

## Related Linear tickets, Github issues, and Community forum posts

https://github.com/n8n-io/n8n/pull/10751 (Closed this to avoid moving the Font option into required node parameters to prevent breaking changes) 

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/nodes-base/nodes/EditImage/EditImage.node.ts [hunk 1]
```diff
@@ -1234,15 +1228,23 @@ export class EditImage implements INodeType {
 						// Combine the lines to a single string
 						const renderText = lines.join('\n');
 
-						const font = options.font || operationData.font;
+						let font = options.font as string | undefined;
```
Comment (ShireenMissi) [inline packages/nodes-base/nodes/EditImage/EditImage.node.ts:1231]: we still need operationData.font
```suggestion
						let font = (options.font || operationData.font) as string | undefined;
```

**Review by ShireenMissi — COMMENTED** [review]
I tested manually and it works 🎉
the only visual issue I see is when you add the option it looks like an error although if you run without selecting it works

Screenshot 2024-10-07 at 17 39 42

**Review by ShireenMissi — APPROVED** [review]
LGTM 🎉

**Comment by janober** [comment]
Got released with [`n8n@1.67.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.67.0)

**Pull request merged**